packages/playground/wordpress-builds/build/build.js

```diff
@@ -25,27 +25,56 @@ const parser = yargs(process.argv.slice(2))
 		},
 		force: {
 			type: 'boolean',
-			description: 'Force rebuild even if the version is already downloaded',
+			description:
+				'Force rebuild even if the version is already downloaded',
 			default: process.env.FORCE_REBUILD === 'true',
 		},
 	});
 
 const args = parser.argv;
 
-let latestVersions = await fetch('https://api.wordpress.org/core/version-check/1.7/?channel=beta')
-	.then((res) => res.json())
+let wpVersions = await fetch(
+	'https://api.wordpress.org/core/version-check/1.7/?channel=beta'
+).then((res) => res.json());
 
-latestVersions = latestVersions
-	.offers
-	.filter((v) => v.response === 'autoupdate')
+wpVersions = wpVersions.offers.filter((v) => v.response === 'autoupdate');
 
 let beta = null;
-if (latestVersions[0].current.includes('beta') || latestVersions[0].current.toLowerCase().includes('rc')) {
-	beta = latestVersions[0];
-	latestVersions = latestVersions.slice(1);
+if (
+	wpVersions[0].current.includes('beta') ||
+	wpVersions[0].current.toLowerCase().includes('rc')
+) {
+	beta = wpVersions[0];
+	wpVersions = wpVersions.slice(1);
 }
 
-function toVersionInfo(apiVersion, slug=null) {
+/**
+ * Create a list of the latest patch versions for each major.minor version.
+ *
+ * Sometimes the API may include multiple patch versions for the same major.minor version.
+ * Playground builds only the latest patch version.
+ */
+const latestVersions = wpVersions.reduce((versionAccumulator, wpVersion) => {
+	const [major, minor] = wpVersion.version.split('.');
+	const majorMinor = `${major}.${minor}`;
+
+	const currentVersionIndex = versionAccumulator.findIndex((v) =>
+		v.version.startsWith(majorMinor)
+	);
+	if (-1 === currentVersionIndex) {
+		return [...versionAccumulator, wpVersion];
+	} else if (
+		semver.gt(
+			wpVersion.version,
+			versionAccumulator[currentVersionIndex].version
+		)
+	) {
+		return versionAccumulator.splice(currentVersionIndex, 1, wpVersion);
```

Comment:
Because [`Array#splice()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) returns an array of the deleted items, I believe this will replace the versionAccumulator with an empty array.

It looks like [`Array#toSpliced()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSpliced) is now a thing that should do what we want with a single call rather than needing to
```js
arr.splice();
return arr;
```

But I guess we could just reassign at the current index and return without creating a new array. Whatever you want... 😄

Reply:
> Because [Array#splice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) returns an array of the deleted items, I believe this will replace the versionAccumulator with an empty array.

Correction:
Because [Array#splice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) returns an array of the deleted items, I believe this will replace the versionAccumulator with an single-item array containing the deleted item.

Example:
```js
[ 1, 2, 3 ].splice(1, 1, 456) // returns [ 2 ]
```

Reply:
Thanks, good catch! Fixed in 3a98021e96a67f2807b4071ca26cad4e2cd8e1cc